main/ui/src/main/java/org/cryptomator/ui/traymenu/TrayIconController.java

```diff
@@ -22,6 +23,8 @@
 	private final Optional<UiAppearanceProvider> appearanceProvider;
 	private final TrayMenuController trayMenuController;
 	private final TrayIcon trayIcon;
+	private boolean initialized;
```

Comment:
does var `initialized` needs to be volatile ?(e.g. set by one thread, but read by another)